utils/atomic-storybook/.prettierrc.js

```diff
@@ -1,5 +1,6 @@
 /** @type {import('prettier').Config} */
 module.exports = {
+  plugins: ['@trivago/prettier-plugin-sort-imports'],
```

Comment:
Fixes a bug where the sort imports was not working

Reply:
Was looking at this couple of days back, great that you are adding it! Is it being added in the main ui-kit prettier config too? 
Also another thing that would be useful to add (in another PR probably) is https://www.npmjs.com/package/eslint-plugin-unused-imports - but I am not sure if the unused vars rule currently catches unused imports too.

Reply:
> Was looking at this plugin couple of days back, great that you are adding it!

Actually it was @btaillon-coveo that added it a while back. It was just broken for some reason. There was rules already in the .prettier config file and they were throwing errors because it the plugin was not explicitly stated. https://github.com/coveo/ui-kit/blob/KIT-2809/utils/atomic-storybook/.prettierrc.js#L23-L24

> Also another thing that would be useful to add (in another PR probably) is https://www.npmjs.com/package/eslint-plugin-unused-imports - but I am not sure if the unused vars rule currently catches unused imports too.

You are right, the unused vars cover this case. I just tested !
<img width="809" alt="image" src="https://github.com/coveo/ui-kit/assets/78121423/3e353aaf-ac15-4d4c-8d6d-7182fe98bc55">